feature-libs/tracking/schematics/add-tracking/index.ts

```diff
@@ -60,13 +196,13 @@ function addPersonalizationFeature(
   });
 }
 
-function addPersonalizationPackageJsonDependencies(packageJson: any): Rule {
+function addTrackingPackageJsonDependencies(packageJson: any): Rule {
   const spartacusVersion = `^${getSpartacusSchematicsVersion()}`;
   const dependencies: NodeDependency[] = [
     {
       type: NodeDependencyType.Default,
       version: spartacusVersion,
-      name: SPARTACUS_PERSONALIZATION,
+      name: SPARTACUS_TRACKING,
```

Comment:
Do we need to specify dependency here?

Option with
```
"ng-add": {
    "save": "dependencies"
  },
```

imho is cleaner

Reply:
Agreed, we don't have to add this dep manually.
We don't even need 
```json
"ng-add": {
    "save": "dependencies"
  },
```
in the tracking's `package.json.`